samples/mobile/src/main/java/com/gluonhq/maps/samples/mobile/MobileSample.java

```diff
@@ -68,7 +70,28 @@
 
     @Override
     public void start(Stage stage) {
-        MapView view = new MapView();
+        TileRetriever osm=new CachedOsmTileRetriever() {
```

Comment:
If this for demo only, why don't we already provide this implementation as part of `com.gluonhq.maps.tile.osm`, exactly as is? Or with a name like `DefaultCachedOsmTileRetriever` ?

Reply:
Well I think it is better when tile provider is explicit, i.e. the developer should know where the tiles are coming from. Also it makes it easy to modify to a different tile provider

Reply:
Well I think it is better if tile provider is explicit, i.e. the developer should know where the tiles are coming from. Also like this it is easy to modify the url to switch to a different provider.